pkg/params/params.go

```diff
@@ -91,7 +90,10 @@ func (p *ParamDesc) GetTitle() string {
 }
 
 func (p *ParamDesc) ToParam() *Param {
-	return &Param{ParamDesc: p}
+	return &Param{
+		ParamDesc: p,
+		value:     p.DefaultValue,
```

Comment:
Nice. This fixes the issue I was hitting regarding default value not being set [here](https://github.com/inspektor-gadget/inspektor-gadget/blob/michael/gadget-registry-local/pkg/gadgets/top/ebpf/tracer/tracer.go#L442) causing panics!

Reply:
Oh yes: I think it's the same as https://github.com/inspektor-gadget/inspektor-gadget/pull/1281/files#r1089072222

Reply:
Just tested it, `runtimes` flags are working fine as well!